README.md

```diff
@@ -4,10 +4,41 @@ A 'package manager' for Glorious Eggroll's custom proton build.
 
 ## Installation
 
+There are currently a couple of ways to install `humble-lumpia`.
+
+**NOTE:** Installing `humble-lumpia` and its dependencies is not sufficient to 
+run the versions of `proton-ge-custom` that `humble-lumpia` will download. Be 
+sure to reference the [README](https://github.com/GloriousEggroll/proton-ge-custom/)
+in `proton-ge-custom` to determine what other dependencies your system requires.
+
+### AUR Package Installation
+
+If you are running **Arch Linux**, you can also download and install the 
+package maintained by [NINNiT](https://github.com/NINNiT) from the AUR. 
+If you are unfamiliar with installing packages from the AUR, you can 
+review the process [here](https://wiki.archlinux.org/title/Arch_User_Repository#Installing_and_upgrading_packages)
+in the Arch Wiki.
+
+### From Source
+
+It is also possible to easily install the utility from source. If you wish to 
+do so, please ensure that you have the appropriate dependencies to 
+[build](#build-dependencies) and [run](#runtime-dependencies) the application 
+as mentioned below.
+
 ```shell
```

Comment:
Maybe put this in it's own `#### Installation` section now? It might also be valuable to put the dependencies sections above the installation section, as you need to have those first.

Reply:
I wasn't 100% as to what you suggested here but I tried to reorder the source installation section to show dependencies required before the installation command. Let me know if that addressed your concern or if there is something else you see that can be improved.